src/app/_shared-components/PostDetails/VoteReferendum/VoteReferendum.tsx

```
@@ -260,11 +286,47 @@ function VoteReferendum({ index, track, onClose, proposalType }: { index: string
-										label={t('VoteReferendum.lockBalance')}
-										onChange={({ value }) => setBalance(value)}
-									/>
+									<div className='flex flex-col gap-y-1'>
+										<BalanceInput
+											name={`${voteDecision}-balance`}
+											label={t('VoteReferendum.lockBalance')}
+											onChange={({ value }) => {
+												setBalance(value);
+												setReuseLock(null);
+											}}
+											value={reuseLock && reuseLock.gt(BN_ZERO) ? reuseLock : undefined}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)